src/main/ui/src/components/SearchByName/SearchByName.jsx

```diff
@@ -1,5 +1,10 @@
 import React from 'react';
```

Comment:
Let's take the `searchByName` changes out. It's still broken and displaying duplicates. We can make the work a different branch that you can tackle next. But make the changes on everything else and we can then move to styling `searchByName`.